### ISSUE TITLE
Biometric no longer working for quicklock 1.08-r01

### ISSUE THREAD

**Issue by smjohns** [PhilippC/keepass2android#1363]
Hi,

I've noticed that when performing a full database unlock the biometric function is working ok but when quicklock is being used the biometric function is being ignored and you have to enter the passcode. 

This used to work but has stopped for some reason?

Device is S7 running Android 8 with all latest security patches.

**Comment by PhilippC** [comment]
what happens? If you don't see the biometric prompt, can you try to tap the fingerprint icon? did you try to re-enable fingerprint unlock in the settings?

**Comment by smjohns** [comment]
Yes for some reason on the quick unlock page I was not seeing the biometric prompt pop-up but was seeing it on full database open page. I checked the settings and it was set to use biometric on all login types. I have tried changing settings but this did not get things moving. 

I tried your suggestion of pressing the fingerprint icon and this did launch the biometric prompt, which then accepted my fingerprint. I tried several reboots of the phone but this did not resolve the issue.

I have today completely uninstalled the app and reinstalled it and after linking it to my DB and re-enabling biometric, it seems to be behaving itself again. Perhaps something got a bit screwey with my phone?
